app/components/publications/publication-flow-selector.js

```diff
@@ -16,23 +16,54 @@ export default class PublicationsPublicationFlowSelectorComponent extends Compon
   *debouncedSearch(searchText) {
     yield timeout(300);
 
-    const filter = {
-      // multiple reference documents on 1 publication-flow must belong to the same case
+    // Filter on all publication-flows linked to the same case
+    // because multiple reference documents on 1 publication-flow must belong to the same case
+    const filterByCase = {
       case: {
         ':id:': this.args.case.get('id'),
       },
     };
+
+    // Filter on all publication-flows 'niet via MR'
+    // to support relinking a publication-flow that was created before
+    // the (sub)case was handled on an agenda
+    const filterNotViaCouncilOfMinisters = {
+      'agenda-item-treatment': {
+        ':has:agendaitem': 'yes',
+      },
+    };
+
     if (!isBlank(searchText)) {
-      filter.identification = {
-        'id-name': searchText,
-      };
+      [filterByCase, filterNotViaCouncilOfMinisters].forEach((filter) => {
+          filter.identification = {
+            'id-name': searchText,
+          };
+      });
     }
-    const publicationFlows = yield this.store.query('publication-flow', {
-      filter,
+
+    // We need to do both queries seperately and combine the results afterwards,
+    // since multiple filters can not be combined using 'OR' in mu-cl-resources
+    const searchByCase = this.store.query('publication-flow', {
+      filter: filterByCase,
       'page[size]': PAGE_SIZE.PUBLICATION_FLOWS,
       sort: 'identification.id-name',
       include: 'identification',
     });
-    return publicationFlows;
+    const searchNotViaCouncilOfMinisters = this.store.query('publication-flow', {
+      filter: filterNotViaCouncilOfMinisters,
+      'page[size]': PAGE_SIZE.PUBLICATION_FLOWS,
+      sort: 'identification.id-name',
+      include: 'identification',
+    });
+
+    const [publicationsByCase, publicationsNotViaCouncilOfMinisters] = yield all([
```

Comment:
Wat is de reden dat nu EmberConcurrency.all wordt gebruikt i.p.v. Promise.all?

Reply:
No specific reason since the passed promises aren't ember-concurrency tasks. Replaced with Promise.all().